src/dosext/mfs/lfn.c

```diff
@@ -102,26 +102,26 @@ void close_dirhandles(unsigned psp)
 
 static int vfat_search(char *dest, char *src, char *path, int alias)
 {
-	struct mfs_dir *dir = dos_opendir(path);
```

Comment:
I won't say that 2-spaces
indent is an "official one"
in dosemu2. Its definitely
inherited from dosemu1
though, and if you like it -
oh well.

Reply:
But I thought we'd agreed on it here #514 and here https://github.com/dosemu2/dosemu2/blob/devel/CONTRIBUTING.md ?

Reply:
Ah, maybe. :)
In any case, its not always
followed. Its just that
reformatting the entire code
base to something else would
be stupid.
Anyway, I am not against your patch.